core/tests/protractor_desktop/emailDashboard.js

```diff
@@ -0,0 +1,66 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview End-to-end tests for email dashboard page.
+ */
+
+var PreferencesPage = require('../protractor_utils/PreferencesPage.js');
+const { browser } = require('protractor');
+var action = require('../protractor_utils/action.js');
+var users = require('../protractor_utils/users.js');
+var waitFor = require('../protractor_utils/waitFor.js');
+
+describe('Email Dashboard', function() {
+  EMAIL_DASHBOARD_URL = '/emaildashboard';
+  var preferencesPage = null;
+
+  beforeAll(async function() {
+    preferencesPage = new PreferencesPage.PreferencesPage();
+    await users.createUser('userA@test.com', 'userA');
+    await users.login('userA@test.com');
+    await preferencesPage.get();
+    await waitFor.pageToFullyLoad();
+    await preferencesPage.toggleEmailUpdatesCheckboxCheckbox();
+    await users.createAndLoginAdminUser(
+      'management@emaildashboard.com', 'management');
+    await browser.get(EMAIL_DASHBOARD_URL);
+    await waitFor.pageToFullyLoad();
+  });
+
+  it('should query for users', async function() {
+    var schemaEditorElement = element(
+      by.css('.protractor-test-email-dashboard-input-3'));
+    var inputElement = schemaEditorElement.element(by.tagName('input'));
+    var submitButton = element(by.css('.protractor-test-submit-query-button'));
+
+    await action.clear('Email dashboard input 3', inputElement);
+    await action.sendKeys('Email dashboard input 3', inputElement, 1);
+    await action.click('Submit Query button', submitButton);
+
+    // Wait 20 seconds for the job to complete.
+    await browser.sleep(20000);
```

Comment:
This test might be flaky, what I have here is a temporary implementation suggested by @kevintab95. Once the initial check is done, I will be discussing on how to implement this test.

Reply:
If this can be flaky, I think we should not add this test. Also, 20 s wait time is not ideal anyway.

Reply:
> ..Once the initial check is done, I will be discussing on how to implement this test.

What do you mean?

Reply:
I meant that I had used sleep for now, once I here suggestions from you and chris, I will be changing the test. I actually don't think this might flake with 20s, but as you said 20s is not ideal also.